packages/babel-plugin-fbt/bin/FbtCollector.js

```diff
@@ -17,12 +17,13 @@ const {SyntaxPlugins} = require('fb-babel-plugin-utils');
 const fs = require('graceful-fs');
 
 /*::
-import type {BabelPluginList} from '@babel/core';
+import type {BabelPluginList, BabelPresetList} from '@babel/core';
```

Comment:
Noticed that we need a Flow type definition for `BabelPresetList`, that is probably similar to this:

https://github.com/facebook/fbt/blob/22c6e3ad79a04e5cd7210c5bb0cd48e6447b1b10/flow-types/nonfb/libdef/flow-typed/babelTypeShims.js#L35-L37